src/main/java/org/hydev/mcpm/utils/ConsoleUtils.java

```diff
@@ -10,6 +12,7 @@
  */
 public class ConsoleUtils
 {
+    public static final PrintStream RAW_OUT = new PrintStream(new FileOutputStream(FileDescriptor.out));
```

Comment:
why this when you could say System.out

Reply:
because `System.out` is patched in Minecraft. it redirects to Minecraft's logger which adds a timestamp instead of the actual System.out that just prints raw text

Reply:
Or actually we can use system.err and avoid this problem

Reply:
But there might be a chance that System.err is patched as well so... I guess this is the safest way

Reply:
I thought System.out used FileDescriptor.out by default but ig making it explicit doesn't hurt